clients/githubrepo/client.go

```diff
@@ -27,19 +27,21 @@ import (
 	"github.com/ossf/scorecard/clients"
 )
 
-const repoFilename = "./githubrepo.tar.gz"
+const repoFilename = "/tmp/%v.%v.githubrepo.tar.gz"
```

Comment:
The right way to create temp dir https://golang.org/pkg/io/ioutil/#TempDir

Reply:
thanks for the link. I was about to make this change to be OS-agnostic.